Uno.Gallery/Uno.Gallery.Shared/Views/SamplePages/ShadowContainerSamplePage.xaml

```diff
@@ -956,15 +956,9 @@
 								<RowDefinition Height="Auto" />
 								<RowDefinition Height="Auto" />
 							</Grid.RowDefinitions>
-							<ItemsControl Background="Green"
-										  x:Name="ShadowsItemsControl">
-								<ItemsControl.ItemsPanel>
-									<ItemsPanelTemplate>
-										<StackPanel Orientation="Vertical"
-													Spacing="24" />
-									</ItemsPanelTemplate>
-								</ItemsControl.ItemsPanel>
-								<ItemsControl.ItemTemplate>
+							<ItemsRepeater Background="Green"
```

Comment:
We can remove the Green background, it was for testing and I forgot to remove it